### PR TITLE
Update, usage examples

### PR DESCRIPTION
**Pull request by CatalinStratu**
Hi,
I've updated the usage examples. I added an example rdf file (**input.rdf**) and an example XML file. I handled the errors where necessary.

Error #3  occurred due to untreated errors. In this PR I have dealt with all the errors in the usage examples.

### CODE REVIEW

**Comment by CatalinStratu** [comment]
@RishabhBhatnagar, what do you think?

**Comment by RishabhBhatnagar** [comment]
Adding a file wouldn't be necessary. If we look at [L32](https://github.com/spdx/gordf/blob/7098f93598fbcdae56f357650dba79c2f8348f34/examples/4-rdfwriter/exampleRDFWriter.go#L32), and the line above, xmlReader is being created from string as well as file.

We can comment out L32 to resolve the issue.

**Comment by CatalinStratu** [comment]
@RishabhBhatnagar 
Yes, that's a good idea. I commented on line 32.

**Comment by CatalinStratu** [comment]
Hi @RishabhBhatnagar, solved!

File: examples/2-rdfparser/exampleRDFParser.go [hunk 1]
```diff
@@ -24,18 +24,20 @@ func main() {
 	xmlReader, err := xmlreader.XMLReaderFromFilePath(filePath)
 	if err != nil {
 		// error reading the rdf file.
-		fmt.Println(err.Error())
+		fmt.Printf("Error reading the rdf file %v: %v", filePath, err)
```
Comment (RishabhBhatnagar) [inline examples/2-rdfparser/exampleRDFParser.go:27]: It is a general convention that the error string must not be capitalized in golang. Other parts of the codebase has followed it. Would be better if it is consistent.

File: examples/2-rdfparser/exampleRDFParser.go [hunk 2]
```diff
@@ -24,18 +24,20 @@ func main() {
...
 		// error parsing the xml content
-		fmt.Println("error parsing the xml content of the rdf file")
+		fmt.Printf("Error parsing the xml content of the rdf file")
+		os.Exit(1)
 	}
 	rdfParser := parser.New()
 	err = rdfParser.Parse(rootBlock)
 	if err != nil {
-		fmt.Printf("error parsing file: %v\n", err)
+		fmt.Printf("Error parsing file: %v\n", err)
```
Comment (RishabhBhatnagar) [inline examples/2-rdfparser/exampleRDFParser.go:39]: same comment here.

File: examples/4-rdfwriter/exampleRDFWriter.go [hunk 3]
```diff
@@ -53,5 +59,9 @@ func main() {
 	var b bytes.Buffer
 
 	// the output will be written to the buffer.
-	rdfwriter.WriteToFile(&b, rdfParser.Triples, rdfParser.SchemaDefinition, tab)
+	err = rdfwriter.WriteToFile(&b, rdfParser.Triples, rdfParser.SchemaDefinition, tab)
+	if err != nil {
+		fmt.Printf("Write to file error: %v\n", err)
```
Comment (RishabhBhatnagar) [inline examples/4-rdfwriter/exampleRDFWriter.go:64]: same here

File: examples/4-rdfwriter/exampleRDFWriter.go [hunk 4]
```diff
@@ -43,7 +48,8 @@ func main() {
 	tab := "    "
 	opString, err := rdfwriter.TriplesToString(rdfParser.Triples, rdfParser.SchemaDefinition, tab)
 	if err != nil {
-		panic("error in a valid example")
+		fmt.Printf("Error in a valid example: %v\n", err)
```
Comment (RishabhBhatnagar) [inline examples/4-rdfwriter/exampleRDFWriter.go:51]: same here.

File: examples/sample-docs/rdf/input.rdf [hunk 5]
```diff
@@ -0,0 +1,1365 @@
+<rdf:RDF
```
Comment (RishabhBhatnagar) [inline examples/sample-docs/rdf/input.rdf:1]: If possible, can we use a smaller rdf file? This file would increase the cloning size of the repository if it is included in the history.

**Review by RishabhBhatnagar — CHANGES_REQUESTED** [review]

File: examples/2-rdfparser/exampleRDFParser.go (hunk 1 above)
Comment (CatalinStratu) [reply examples/2-rdfparser/exampleRDFParser.go]: I made the necessary changes

File: examples/2-rdfparser/exampleRDFParser.go (hunk 2 above)
Comment (CatalinStratu) [reply examples/2-rdfparser/exampleRDFParser.go]: I made the necessary changes

File: examples/4-rdfwriter/exampleRDFWriter.go (hunk 3 above)
Comment (CatalinStratu) [reply examples/4-rdfwriter/exampleRDFWriter.go]: I made the necessary changes

File: examples/4-rdfwriter/exampleRDFWriter.go (hunk 4 above)
Comment (CatalinStratu) [reply examples/4-rdfwriter/exampleRDFWriter.go]: I made the necessary changes

File: examples/sample-docs/rdf/input.rdf (hunk 5 above)
Comment (CatalinStratu) [reply examples/sample-docs/rdf/input.rdf:1]: I added a smaller file.

**Review by RishabhBhatnagar — APPROVED** [review]

**Comment by RishabhBhatnagar** [comment]
@CatalinStratu , Thanks for quickly making the changes. 
There was one other small change. But I can handle that on my own.

Thanks for overall effort and patience with this issue.

**Pull request merged**